### ISSUE TITLE
IOS requires twice to press actions

### ISSUE THREAD

**Issue by atomoc** [Adphorus/react-date-range#241]
Try to select the date in the IOS simulator and make sure of it. At the same time I tried everything, not having access to the kernel did not solve the problem.

**Comment by Maggistro** [comment]
I have the same behaviour. 

I started to dig around a bit using weinre and found out some things about the way the clicks add classes.
- First Click:
 - Classes "rdrDayStartPreview rdrDayEndPreview" are added to day
- Second Click ( Same date ):
 - Classes change to "rdrStartEdge rdrEndEdge".
- Second Click ( different date )
 - Classes get removed from first date
 - Classed "rdrStartEdge rdrEndEdge" get added to different date

If one continues to select end dates, the calender will select the end Date with the same behavoir. It first triggers the "rdrDayEndPreview" class and on second click remove / replace this with "rdrEndEdge".

Would like to see this fixed. Currently i do not have the time to fix it myself.

**Comment by Maggistro** [comment]
Just noticed. Behaviour can be fixed by setting showPreview Option to false -.-. fml

**Comment by Maggistro** [comment]
Yes Problem is, this will disable the preview mode for ALL devices, so no hover effect on desktops.
Can be done by only using preview in mobile with breakpoints, but a real fix would be to handle the iphones pseudo hover effect correctly.

If theres time i will look into it.

**Comment by huangliop** [comment]
Component Calendar have same issue and showPreview cant fix it
